classes/PolicyPositions.php

```diff
@@ -111,8 +111,7 @@ private function getMemberPolicyPositions ($limit = NULL) {
                     $this->positions[] = array( 'policy_id' => $policy[0], 'desc' => $dream_info['full_sentence'] );
                     $this->positionsById[$policy[0]] = array(
                         'policy_id' => $policy[0],
-                        'desc' => $dream_info['full_sentence'],
-                        'voted' => $dream_info['voted']
+                        'desc' => $dream_info['full_sentence']
```

Comment:
Probably not wanting to be done here, but just to note I'm not sure why here we have both `$positions` and `$positionsById` which (after this PS) contain exactly the same information.